packages/@aws-cdk/aws-scheduler-alpha/lib/group.ts

```diff
@@ -0,0 +1,363 @@
+    return this.metric('InvocationThrottleCount', props);
+  }
+
+  /**
+   * Metric for all invocation attempts.
+   *
+   * @default - sum over 5 minutes
+   */
+  public metricAttempts(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+    return this.metric('InvocationAttemptCount', props);
+  }
+
+  /**
+   * Emitted when the target returns an exception after EventBridge Scheduler calls the target API.
+   *
+   * @default - sum over 5 minutes
+   */
+  public metricTargetErrors(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+    return this.metric('TargetErrorCount', props);
+  }
+
+  /**
+   * Metric for invocation failures due to API throttling by the target.
+   *
+   * @default - sum over 5 minutes
+   */
+  public metricTargetThrottled(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+    return this.metric('TargetErrorThrottledCount', props);
+  }
+
+  /**
+   * Metric for dropped invocations when EventBridge Scheduler stops attempting to invoke the target after a schedule's retry policy has been exhausted.
+   *
+   * @default - sum over 5 minutes
+   */
+  public metricDropped(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+    return this.metric('InvocationDroppedCount', props);
+  }
+
+  /**
+   * Metric for invocations delivered to the DLQ
+   *
+   * @default - sum over 5 minutes
+   */
+  metricSentToDLQ(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+    return this.metric('InvocationsSentToDeadLetterCount', props);
+  }
+
+  /**
+   * Metric for failed invocations that also failed to deliver to DLQ.
+   *
+   * @default - sum over 5 minutes
+   */
+  metricFailedToBeSentToDLQ(errorCode?: string, props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+    if (errorCode) {
+      return this.metric(`InvocationsFailedToBeSentToDeadLetterCount_${errorCode}`, props);
+    }
+
+    return this.metric('InvocationsFailedToBeSentToDeadLetterCount', props);
+  }
+
+  /**
+   * Metric for delivery of failed invocations to DLQ when the payload of the event sent to the DLQ exceeds the maximum size allowed by Amazon SQS.
+   *
+   * @default - sum over 5 minutes
+   */
+  metricSentToDLQTrunacted(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+    return this.metric('InvocationsSentToDeadLetterCount_Truncated_MessageSizeExceeded', props);
+  }
+
+  /**
+   * Grant the indicated permissions on this group to the given principal
+   */
+  grant(grantee: iam.IGrantable, ...actions: string[]): iam.Grant {
+    return iam.Grant.addToPrincipal({
+      grantee,
+      actions,
+      resourceArns: [this.groupArn],
+      scope: this,
+    });
+  }
+
+  arnForScheduleInGroup(scheduleName: string): string {
+    return Arn.format({
+      region: this.env.region,
+      account: this.env.account,
+      partition: Aws.PARTITION,
+      service: 'scheduler',
+      resource: 'schedule',
+      resourceName: this.groupName + '/' + scheduleName,
+    });
+  }
```

Comment:
Not sure if this is right, will check with @Jacco  how to test it better without implemented integration tests. 

Also, should we make this method private?

Reply:
Lets add a good integration in the next PR.

Reply:
Think it can be private.